setup.py

```diff
@@ -1,9 +1,14 @@
 from setuptools import setup, find_packages
 
 
+# Do not import any validator modules here, as that would require a correct
+# dependency environment that setup.py is not guaranteed to have.
+execfile("validator/_version.py")
+
+
```

Comment:
I'd rather have something simpler and more widespread like storing the version in the `__init__.py` file, as a `VERSION` or `__version__`.

Here's an example:
https://github.com/django/django/blob/master/setup.py#L32
and https://github.com/django/django/blob/master/django/__init__.py#L3

I wouldn't bother reimplementing the `get_version` though, I think we can manually update both the `VERSION` and `__version__` on each release.